lib/paynet_api/client.rb

```diff
@@ -21,7 +21,7 @@ def search_for_company(params)
 
     private
       def get request
-        connection = Faraday.new request.url do |conn|
+        connection = Faraday.new(request.url, { proxy: request.options[:proxy_url] } ) do |conn|
```

Comment:
i think `request.proxy_url` would work here

Reply:
@drewnichols yes, it did just tested it